view/rooms/app.js

```diff
@@ -931,6 +930,8 @@ function pageImcTeam() {
     ],
   };
   u.postPromiseWithUrl(GET_ACTIONS, serviceReq);
+  closePagingModal();
+  alert('Thanks, IMC team was paged and it’s working hard to fix the issue');
```

Comment:
(I should have mentioned in the last comment) The message sounds we are already committed to fix the issue at the moment the page sent out. 
How about 'Thanks, IMC team was paged. We will look into it ASAP' or simply 'Thanks, IMC team was paged.'? 
What do others think?

Reply:
I agree, maybe the `Thanks, IMC team was paged. We will look into it ASAP` which has a sound more along the lines of setting the expectation. Worth hearing what other think.